### DISCUSSION TITLE
What is `fzf` plugin supposed to do exactly? I didn't see any change after enabling it in `.zshrc`.

### DISCUSSION THREAD

**Discussion by a-pav** [ohmyzsh/ohmyzsh#11536] [Help] [answered]
I was looking for a way to get auto-completion wherever I type `fzf --<Tab>`, so I enabled the `fzf` plugin. But whether it's enabled or not, I don't get any auto-completion like that. Though in either case, fzf's default keybindings (CTRL_T, CTRL_R and ALT_C) work as expected.

This is the relevant part of my `.zshrc` file:
```sh
plugins=(
    fzf
    adb
    # aliases
    # colored-man-pages
    extract
    fd
    git
    golang
    man
    sudo
    timer
    zsh-autosuggestions
)

# fzf
if [[ -f ~/.fzf.zsh ]]; then
    source ~/.fzf.zsh
fi

if [[ -f ~/.fzfrc ]]; then
    source ~/.fzfrc  # I initialize and export fzf-related variables inside the `~/.fzfrc` file
fi

export FZF_BASE="$HOME/.fzf"
DISABLE_FZF_AUTO_COMPLETION="false"
DISABLE_FZF_KEY_BINDINGS="false"

source "$ZSH/oh-my-zsh.sh"
```

What am I missing? What is the expected behaviour of `fzf` plugin? I wonder if it's something like [`fzf-tab`](https://github.com/Aloxaf/fzf-tab) project or it just provides ordinary auto-completion for the `fzf` command only?

**Comment by carlosala** [comment, marked as answer]
It sets up the keybindings, looks where is installed, etc. Those are things that you are doing probably on your `.fzf.zsh`.

**Reply by a-pav** [reply]
Does it mean that I don't need to enable this plugin if my `fzf` setup already works as expected?

**Reply by carlosala** [reply]
probably not, you can dive into the code to see if you are missing something

**Reply by a-pav** [reply]
I think I'm good. Thanks anyways.
